### PR TITLE
made progress on API for Stereoscope

### LINKED ISSUE #888: Enhance Stereoscope API
1. Remove the call to .get_params in the workflow
2. Move parameters to default entries (especially learning rate etc..)
3. Return nice dataframes for proportions

### PR DESCRIPTION
**Pull request by romain-lopez**
closes #888

### CODE REVIEW

**Comment by romain-lopez** [comment]
@adamgayoso, do you know what's wrong with the docs?

File: scvi/external/stereoscope/_model.py [hunk 1]
```diff
@@ -130,8 +175,33 @@ def __init__(
...
             st_adata.n_obs,
         )
+        self.cell_type_mapping = cell_type_mapping
         self.init_params_ = self._get_init_params(locals())
 
+    @classmethod
+    def from_rna_model(
+        cls,
+        st_adata: AnnData,
+        sc_model: RNAStereoscope,
```
Comment (adamgayoso) [inline scvi/external/stereoscope/_model.py:185]: should add the other kwargs here

File: scvi/external/stereoscope/_module.py [hunk 2]
```diff
@@ -129,22 +129,23 @@ class SpatialDeconv(AbstractVAE):
     ----------
     n_spots
         Number of input spots
-    params
-        Tuple of ndarray of shapes [(n_genes, n_labels), (n_genes)] containing the dictionnary and log dispersion parameters
+    sc_model
```
Comment (adamgayoso) [inline scvi/external/stereoscope/_module.py:132]: sc_params

File: scvi/external/stereoscope/_module.py [hunk 3]
```diff
@@ -129,22 +129,23 @@ class SpatialDeconv(AbstractVAE):
...
+        Model learned on the single-cell RNA sequencing data
```
Comment (adamgayoso) [inline scvi/external/stereoscope/_module.py:133]: scparams description

**Review by adamgayoso — CHANGES_REQUESTED** [review]

File: scvi/external/stereoscope/_model.py (hunk 1 above)
Comment (romain-lopez) [reply scvi/external/stereoscope/_model.py:185]: yeah the kwargs got me!

Comment (adamgayoso) [reply scvi/external/stereoscope/_model.py:185]: as well as docstring descriptions

**Comment by romain-lopez** [comment]
Alright, I tested on my reproducibility notebooks and it works. Let's wait for this, but it can be merged afterwards.

**Review by adamgayoso — APPROVED** [review]

**Pull request merged**